src/importexport/guitarpro/internal/importgtp.cpp

```diff
@@ -2994,6 +2994,9 @@ Score::FileError importGTP(MasterScore* score, const QString& name)
     //
     std::vector<Part*> infoParts;
     for (Part* part : score->parts()) {
+        //! HACK Temporary disabled, something not corrected, so crashed
+        continue;
+
```

Comment:
Is this hack still needed? That "temporary" meanwhile lasts more than half a year...